extensions/vertx/runtime/src/main/java/io/quarkus/vertx/ConsumeEvent.java

```diff
@@ -63,7 +66,7 @@
      * @return {@code true} if the address should not be propagated across the cluster
      * @see io.vertx.core.eventbus.EventBus#localConsumer(String)
      */
-    boolean local() default false;
+    boolean local() default true;
```

Comment:
Hmmm, you change the default value here. Is it something we want? If so, it should at least be a separate commit if not a separate PR.

Reply:
it's explicity ask in  the ticket: #2529

"At the same time, the local attribute of ConsumeEvent should be passed to true by default."